spec/features/participants_spec.rb

```diff
@@ -47,48 +47,40 @@
   scenario "logged in as an Organizer I want to be able to sort the participants table by their eating habits" do
     user = FactoryGirl.create(:user)
     profile = FactoryGirl.create(:profile, user: user, last_name: "Peter")
-    application_letter = FactoryGirl.create(:application_letter_accepted, 
+    application_letter = FactoryGirl.create(:application_letter_accepted,
       user: user, event: @event, vegan: true)
     user = FactoryGirl.create(:user)
     profile = FactoryGirl.create(:profile, user: user, last_name: "Paul")
-    application_letter = FactoryGirl.create(:application_letter_accepted, 
+    application_letter = FactoryGirl.create(:application_letter_accepted,
       user: user, event: @event, vegan: true, allergic: true)
     user = FactoryGirl.create(:user)
     profile = FactoryGirl.create(:profile, user: user, last_name: "Mary")
-    application_letter = FactoryGirl.create(:application_letter_accepted, 
+    application_letter = FactoryGirl.create(:application_letter_accepted,
       user: user, event: @event, vegetarian: true)
     user = FactoryGirl.create(:user)
     profile = FactoryGirl.create(:profile, user: user, last_name: "Otti")
-    application_letter = FactoryGirl.create(:application_letter_accepted, 
+    application_letter = FactoryGirl.create(:application_letter_accepted,
       user: user, event: @event, vegetarian: true, allergic: true)
     user = FactoryGirl.create(:user)
     profile = FactoryGirl.create(:profile, user: user, last_name: "Benno")
-    application_letter = FactoryGirl.create(:application_letter_accepted, 
+    application_letter = FactoryGirl.create(:application_letter_accepted,
       user: user, event: @event)
```

Comment:
How about I add a comment to that block, explaining what that block does?
If that's ok and it's the only thing on this PR you'd like changed, I'd appreciate if this PR would be approved, since it's blocking my other PR and that blocks our Kanban board.